### ISSUE TITLE
Support for key resharing in GG2020

### ISSUE THREAD

**Issue by tmpfs** [ZenGo-X/multi-party-ecdsa#164]
Does the implementation of GG2020 support key re-sharing?

If so, is there an example of the re-sharing part of the protocol I could take a look at.

Thanks, as always 🙏

**Comment by GTC6244** [comment]
Hey muji -  check out https://github.com/ZenGo-X/fs-dkr !

**Comment by tmpfs** [comment]
Thanks, @GTC6244, that looks like just what I need! It appears to be using an older version of `multi-party-ecdsa` and `curv-kzen`; before I look into using the code i think it would be good to update those dependencies. If I submit a PR for updating `curv` and `multi-party-ecdsa` in `fs-dkr` would somebody here be willing to review/merge it?

/cc @survived

**Comment by GTC6244** [comment]
Yes @tmpfs , it's using GG2018, and, IIRC a custom version of curv-kzen bundled in as code... I believe that the current version of curv-kzen (0.9.0 as of writing) doesn't meet wasm32-unknown-unknown, so there's likely some deeper work to be done.

**Comment by drewstone** [comment]
I'd be happy to add a review too @tmpfs
